docs/1.34/faq/migrate-to-self-hosted-fq15.mdx

```diff
@@ -0,0 +1,100 @@
+
+1. Upgrade your application from Prisma 1 to Prisma ORM. In this case, you will no longer need Prisma 1 Cloud because Prisma ORM does not require a separate Prisma server.
+2. Self-host Prisma 1. While it is a viable option, we do not recommend self-hosting Prisma 1 as it is in maintenance mode, and Prisma will not develop new features or offer ongoing support for issues.
+
+#### Upgrading to Prisma ORM
+
+When upgrading to [Prisma ORM](https://prisma.io), there are two general strategies:
+
+1. **Upgrade all at once**: Remove Prisma 1 from your project and move everything to Prisma ORM all at once.
+2. **Side-by-side upgrade**: Add Prisma ORM to an existing Prisma 1 project and gradually replace existing Prisma 1 features while running each side-by-side.
+
+Learn more about [how to upgrade](https://www.prisma.io/docs/guides/upgrade-guides/upgrade-from-prisma-1/how-to-upgrade#upgrade-strategies), [the main differences between Prisma 1 and Prisma ORM](https://www.prisma.io/docs/guides/upgrade-guides/upgrade-from-prisma-1/how-to-upgrade#main-differences-between-prisma-1-and-prisma-2), and the [Prisma upgrade CLI](https://www.prisma.io/docs/guides/upgrade-guides/upgrade-from-prisma-1/how-to-upgrade#prisma-upgrade-cli).
+
+#### Self-hosting Prisma 1
+
+We do not recommend self-hosting Prisma 1 as it is in maintenance mode, and Prisma will not develop new features or offer ongoing support for issues. The following are high-level steps for self-hosting if it is not feasible to upgrade your application to Prisma ORM in the time before the Prisma 1 Cloud sunset on September 1st, 2022.
+
+1. Deploy a Prisma 1 Server to the hosting environment of your choice. Use the following tutorials as examples:
+1. [Deploy to Kubernetes](https://v1.prisma.io/tutorials/deploy-prisma-to-kubernetes-ct13)
+2. [Deploy to Digital Ocean](https://v1.prisma.io/tutorials/deploy-prisma-to-digitalocean-ct12)
+3. [Deploy to Digital Ocean with Docker](https://v1.prisma.io/tutorials/deploy-prisma-to-digitalocean-with-docker-machine-ct06)
+4. [Deploy to Dokku](https://v1.prisma.io/tutorials/deploy-prisma-to-dokku-ct15)
+2. [Export your service data](https://v1.prisma.io/tutorials/data-import-and-export-with-prisma-cs03) from Prisma 1 Cloud using [`prisma1 export`](https://v1.prisma.io/docs/1.34/prisma-cli-and-configuration/cli-command-reference/prisma-export-ycv2/) or email us using the email address from your account with a list of the **Services** in your account that you would like us to back up.
+3. Update the connection strings in your local `prisma.yml` file to point to the Prisma 1 Server created in Step 2, and [import your service data](https://v1.prisma.io/tutorials/data-import-and-export-with-prisma-cs03).
+4. Regression test your application in pre-production environments, then deploy the application with new connection strings to production.
```

Comment:
If this is not a list of steps to take after each other, let's make this an unordered list:
```suggestion
- Deploy a Prisma 1 Server to the hosting environment of your choice. Use the following tutorials as examples:
- [Deploy to Kubernetes](https://v1.prisma.io/tutorials/deploy-prisma-to-kubernetes-ct13)
- [Deploy to Digital Ocean](https://v1.prisma.io/tutorials/deploy-prisma-to-digitalocean-ct12)
- [Deploy to Digital Ocean with Docker](https://v1.prisma.io/tutorials/deploy-prisma-to-digitalocean-with-docker-machine-ct06)
- [Deploy to Dokku](https://v1.prisma.io/tutorials/deploy-prisma-to-dokku-ct15)
- [Export your service data](https://v1.prisma.io/tutorials/data-import-and-export-with-prisma-cs03) from Prisma 1 Cloud using [`prisma1 export`](https://v1.prisma.io/docs/1.34/prisma-cli-and-configuration/cli-command-reference/prisma-export-ycv2/) or email us using the email address from your account with a list of the **Services** in your account that you would like us to back up.
- Update the connection strings in your local `prisma.yml` file to point to the Prisma 1 Server created in Step 2, and [import your service data](https://v1.prisma.io/tutorials/data-import-and-export-with-prisma-cs03).
- Regression test your application in pre-production environments, then deploy the application with new connection strings to production.
```

Reply:
@janpio  this is a mostly sequential list of steps but I took your advice and made the child list unordered. (with the deploy links)